README.md

```diff
@@ -60,4 +58,19 @@ documentation located in the [`docs/`](docs) directory.
 * Contributions to this project are welcome: if you want to add a feature or a
   fix a bug, please do so by opening a Pull Request in this GitHub repository.
   In case of feature contribution, we kindly ask you to open an issue to
```

Comment:
```suggestion
  In case of feature contribution, we kindly ask you to [open an issue](https://github.com/macstadium/packer-plugin-macstadium-orka/issues) to
```